.github/workflows/charm-update-libs.yaml

```diff
@@ -47,14 +52,14 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Merge any pre-existing PRs from the automatically generated "chore/auto-libs" branch if the CI is green
+      - name: Merge any pre-existing PRs from the automatically generated branch if the CI is green
         run: |
-          # If a PR from "chore/auto-libs" is open and CI checks are passing, merge it
-          is_pr_open="$(gh pr list --head chore/auto-libs --state open --json id --jq 'length')"
+          # If a PR from the chore branch is open and CI checks are passing, merge it
+          is_pr_open="$(gh pr list --head "${{ inputs.git-branch }}" --state open --json id --jq 'length')"
           if [[ "$is_pr_open" == "1" ]]; then
-            if gh pr checks chore/auto-libs; then
-              echo "CI checks are passing, merging the chore/auto-libs PR"
-              gh pr merge chore/auto-libs --admin --squash --delete-branch
+            if gh pr checks "${{ inputs.git-branch }}"; then
+              echo "CI checks are passing, merging the ${{ inputs.git-branch }} PR"
+              gh pr merge "${{ inputs.git-branch }}" --admin --squash --delete-branch
```

Comment:
I would take `${{ inputs.git-branch }}` outside of the steps and have an `env:` key at the job level instead:

```yaml
...
runs-on: ubuntu-22.04
env:
  GIT_BRANCH: "${{ inputs.git-branch }}"
...
```

Then you can use the bash variable "$GIT_BRANCH" in the rest of the workflow :)